apache/superset
# feat: bump superset-ui dependencies to v0.12 and plugins to v0.11

### CATEGORY

Choose one


### SUMMARY
* Upgrade `@superset-ui` packages to `0.12`
  * Some prop names changed for clarity
  * `@superset-ui/query` is split from `@superset-ui/chart`
  * Support for control panel as part of plugin
  * New `durationFormatter` cc: @villebro 
* Upgrade `@superset-ui-plugins` packages to `0.11`
  * to make all plugins compatible with `@superset-ui@0.12`
* Make changes to `filter_box`, `time_table` and `deck.gl` plugins that reside in this main repo to conform to the new APIs. (See migration guide from [this PR](https://github.com/apache-superset/superset-ui-plugins/pull/183))

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

![image](https://user-images.githubusercontent.com/1659771/63722174-7d0f6d00-c807-11e9-9103-e7770e18c164.png)

### TEST PLAN

* Verify that all charts are working like before
* Try setting table filter

### REVIEWERS

@etr2460
